src/app/components/search/SearchResultsTable/StatusCell.js

```diff
@@ -34,12 +34,14 @@ export default function StatusCell({ projectMedia }) {
 }
 StatusCell.propTypes = {
   projectMedia: PropTypes.shape({
-    verification_statuses: PropTypes.shape({
-      statuses: PropTypes.arrayOf(PropTypes.shape({
-        id: PropTypes.string.isRequired,
-        label: PropTypes.string.isRequired,
-      }).isRequired), // undefined during optimistic update
-    }).isRequired,
+    team: PropTypes.shape({
+      verification_statuses: PropTypes.shape({
+        statuses: PropTypes.arrayOf(PropTypes.shape({
+          id: PropTypes.string.isRequired,
+          label: PropTypes.string.isRequired,
+        }).isRequired), // undefined during optimistic update
```

Comment:
This comment confuses me. Why is `verification_statuses` undefined during optimistic update? Can we fix that?

Reply:
This comment was already there. It seems it was added by you in PR #610. I'm not sure what is going on there.

Reply:
Hahaha! Well, that's embarrassing :). I guess it's not worth tracking down what I was thinking....